poll/example_test.go

```diff
@@ -0,0 +1,40 @@
+package poll
```

Comment:
We should the `_test` special package for example so that it is closer to what user would have in their code.
I'll do a follow-up 😉